content/pages/getting_started/what_do_i_need.md

```diff
@@ -0,0 +1,96 @@
+---
+title: What Do I Need?
+description: Read on to learn about the basics you'll need to get started on your blacksmithing journey.
+date: 2023-05-31T18:06:48-05:00
+lastmod: 2024-02-11
+---
+The most common question we get from new blacksmiths is "what equipment do I need to get started?"
+
+This article hopes to answer that very question.
+
+## PPE
+
+These things are *non-optional* no matter your budget.
+
+Eye protection and hearing protection are essential equipment - don't skip this section. Even if you aren't using power tools, extended exposure to the noise associated with hammering can do lasting damage to your hearing. Likewise, bits of flying metal are always a risk in any workshop - make sure this shrapnel hits your eye pro instead of your eyes. Eyeglasses or sunglasses will work, but a pair of dedicated safety glasses or a face shield are better choices. A respirator is generally not required for general forging activities, but is a really good thing to have on hand for [preparing your propane forge for its first use][how to line your forge], burning off harmful compounds, or grinding.
+
+* Eye protection - ANSI-rated [face shield][] or safety glasses
+* Hearing protection - [earplugs][] or muffs
+* Lung protection (for grinding and handling particulates) - P100 or better [respirator][]
+
+## The Basics: Everything you need to get started
+
+The core requirements for a blacksmith's shop are a hammer, an anvil, a forge, and probably some tongs. While it is possible to make tongs without having tongs, there's no shame in buying your first (or first several) set of tongs. If you're on a tight budget, [Ken's Custom Iron tong blanks][] are a great middle-ground option.
```

Comment:
Can we put like some kind of shiny text box that talks about secondhand goods? As you've brought up this week, a lot of people show up trying to squeeze blood out of a stone when it comes to their budget; maybe telling people to keep an eye out for used goods would be a prudent option (if it's already there, ignore me --- if only I could read!)

Reply:
What do we want it to say?